### PR TITLE
feature-slide6,7

### PR DESCRIPTION
**Pull request by DoQuangKhanh0209**
Based on what you’ve learned, update the app to satisfy these requirements:

1. UITableView: Display list of songs
• Cell contains UIImage, and 2 UILabel
2. UINavigationController
3. NavigationController title always in large title
• When tap on a cell, navigate to detail screen
• When tap on a cell, navigate to detail screen
• User are able to control music player even when app is in background and in lock screen
Evidence:

 Image 
E079B57A-6673-4927-BC2C-485F4CEC56D5
C028B220-7B0F-48AD-84D8-9909D8703709

Video
https://user-images.githubusercontent.com/114377716/196615608-08ca0d61-bf07-4797-b8a1-5b559c0ebacd.mp4

### CODE REVIEW

File: AppMusicPlayer/AppMusicPlayer/PlayMediaViewController.swift [hunk 1]
```diff
@@ -50,11 +50,55 @@ final class PlayMediaViewController: UIViewController {
...
+                case.remoteControlNextTrack:
+                    index = ((index ?? 0) + 1 + 3) % 3
+                    setupDataUIMusic()
+                    playSongMusic()
+                case.remoteControlPreviousTrack:
+                    index = ((index ?? 0) - 1 + 3) % 3
+                    setupDataUIMusic()
+                    playSongMusic()
+                default:
+                    print("")
```
Comment (tunghs-1253) [inline AppMusicPlayer/AppMusicPlayer/PlayMediaViewController.swift]: Không để print linh tinh thế này anh Khánh nhé

File: AppMusicPlayer/AppMusicPlayer/SongListViewController.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        return cell
+    }
+}
+
+extension SongListViewController: UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        songListTable.deselectRow(at: indexPath, animated: false)
+        let sb = UIStoryboard(name: "Main", bundle: nil)
+        let playMediaViewController = sb.instantiateViewController(identifier: "PlayMedia") as! PlayMediaViewController
```
Comment (tunghs-1253) [inline AppMusicPlayer/AppMusicPlayer/SongListViewController.swift]: Không force unwrap

File: AppMusicPlayer/DataMusic/ListMusic.swift [hunk 3]
```diff
@@ -9,8 +9,8 @@ import Foundation
 
 class ListMusic {
     static let listSong: [MusicModel] = [
-        MusicModel(name: "Da Lo Yeu Em Nhieu", author: "Justatee", imageMusic: "audio-Image", fileMusic: "song1"),
-        MusicModel(name: "Waiting For You", author: "Mono", imageMusic: "audio2-Image", fileMusic: "song2"),
-        MusicModel(name: "Good Morning", author: "Jack5t", imageMusic: "audio3-Image", fileMusic: "song3"),
+        MusicModel(name: "Da Lo Yeu Em Nhieu", author: "Justatee", imageMusic: "audio-Image", fileMusic: "song2"),
+        MusicModel(name: "Waiting For You", author: "Mono", imageMusic: "mono", fileMusic: "song1"),
+        MusicModel(name: "Chuyen doi ta", author: "Da lab", imageMusic: "chuyendoita", fileMusic: "song3"),
```
Comment (daolq-2712) [inline AppMusicPlayer/DataMusic/ListMusic.swift]:
```suggestion
        MusicModel(name: "Waiting For You", author: "Mono", imageMusic: "mono", fileMusic: "song1"),
        MusicModel(name: "Da Lo Yeu Em Nhieu", author: "Justatee", imageMusic: "audio-Image", fileMusic: "song2"),
        MusicModel(name: "Chuyen doi ta", author: "Da lab", imageMusic: "chuyendoita", fileMusic: "song3"),
```

**Review by daolq-2712 — COMMENTED** [review]
Pls check my comment!

**Review by daolq-2712 — APPROVED** [review]
OK!

**Pull request merged**